src/obb/server/http_component.clj

```diff
@@ -15,19 +15,21 @@
                   [#"^http://localhost(.*)"]
                   :access-control-allow-methods [:get :put :post :delete]))
 
-(def app
+(defn app
   "The main app handler"
+  [system]
   (-> (compojure/routes routes/public-routes)
       (setup-cors)))
 
-(defrecord HttpServerComponent [server port]
+(defrecord HttpServerComponent [server port meta]
 
   component/Lifecycle
 
   (start [component]
     (println (str "** OBB HTTP Server " (or (get (System/getenv) "OBB_ENV" "development"))
                   " running on port " port))
-    (assoc component :server (http/start-server app {:port port})))
+    (assoc component :server (http/start-server (app (:system meta))
+                                                     {:port port})))
```

Comment:
We've spoke about changing this strategy in Slack. I gave this reference as an alternative: https://youtu.be/13cmHf_kt-Q?t=1849

(This comment is just for future reference)